corehq/ex-submodules/pillowtop/feed/couch.py

```diff
@@ -25,11 +26,13 @@ def iter_changes(self, since, forever):
             heartbeat=True,
             since=since,
             filter=self._couch_filter,
-            include_docs=self._include_docs,
+            include_docs=True,
             **extra_args
         )
         for couch_change in changes_stream:
-            yield change_from_couch_row(couch_change, document_store=self._document_store)
+            change = change_from_couch_row(couch_change, document_store=self._document_store)
+            populate_change_metadata(change, SOURCE_COUCH, self._couch_db.dbname)
```

Comment:
populating this here to make it easier to do the retry (also means we don't have to re-fetch the doc when we retry the change)